apps/explorer/src/ui/header/NetworkSelect.tsx

```diff
@@ -255,7 +266,7 @@ export function NetworkSelect({
 										}}
 									/>
 									{!!value && version && binaryVersion ? (
-										<div className="-mx-3 -mb-4 mt-2 rounded-b-lg bg-gray-40">
+										<div className="-mx-3 -mb-4 mt-2 rounded-b-lg bg-gray-40 backdrop-blur-5">
```

Comment:
it doesn't seem like backdrop-blur is working as expected: 

<img width="311" alt="image" src="https://github.com/MystenLabs/sui/assets/122397493/9df24401-bef6-43d7-b8e2-77a4bbe221ac">

Reply:
it looks like the background should be `bg-hero-darkest/5`

Reply:
updated